include/swift/Option/Options.td

```diff
@@ -148,6 +148,10 @@ def F : JoinedOrSeparate<["-"], "F">, Flags<[FrontendOption]>,
   HelpText<"Add directory to framework search path">;
 def F_EQ : Joined<["-"], "F=">, Flags<[FrontendOption]>, Alias<F>;
 
+def Fsystem : JoinedOrSeparate<["-"], "Fsystem">, Flags<[FrontendOption]>,
```

Comment:
Right, I found it surprising that -F= could be joined, I'd prefer if we avoided '=' joined aliases completely, but ship has sailed.